pom.xml

```diff
@@ -42,7 +42,7 @@
         <cxf.compiler.fork>false</cxf.compiler.fork>
         <cxf.build-utils.version>3.4.5-SNAPSHOT</cxf.build-utils.version>
         <cxf.xjc-utils.version>4.0.0-SNAPSHOT</cxf.xjc-utils.version>
-        <cxf.jdk.version>17</cxf.jdk.version>
+        <cxf.jdk.version>11</cxf.jdk.version>
```

Comment:
If I am not mistaken, Spring JARs are build using JDK-17, does 11 work for you?

Reply:
This works. We can use this flag to run tests with JDK-17 and cross compile the JDK-11 cxf artifacts. That means the JDK-11 can still be supported if user doesn't want Spring, and Spring6 and cxf can still work with JDK-17 if user wants to include Spring.

Reply:
👍 thanks, let's merge it like that, we could always change it later. On Jenkins we cross compile with JDK8 / JDK11 / JDK17 and JDK18 / JDK19 , but for `master` we should bump to JDK-11, will do that shortly.